docs/content/en/getting_started/upgrading/2.37.md

```diff
@@ -10,8 +10,27 @@ description: Breaking Change for HELM deployments and MySQL / RabbitMQ users
 Support for MySQL and RabbitMQ was previously deprecated [see discussion post here](https://github.com/DefectDojo/django-DefectDojo/discussions/9690)) and is now being removed
 
 MySQL and RabbitMQ have been removed from the following places:
+
 - Docker Compose files
 - Helm Chart
 - Unit/Integration CI/CD Tests
 
-There are no other special instructions for upgrading to 2.37.x. Check the [Release Notes](https://github.com/DefectDojo/django-DefectDojo/releases/tag/2.36.0) for the contents of the release.
\ No newline at end of file
+Previous HELM `values.yaml` file was not following the official HELM best practicies on key naming - [https://helm.sh/docs/chart_best_practices/values/#naming-conventions](https://helm.sh/docs/chart_best_practices/values/#naming-conventions)
```

Comment:
Because `2.37.0` was just released, these release notes need to be moved to the `2.38.md` file

Reply:
Done

Reply:
I still see some change in `2.37`. I'm just not able to identify what kind of change. Probably some whitespace.

Can you also add "Breaking Change for HELM deployments" or sth similar to `description` (instead of "No special instructions.")

Reply:
Thank you, fixed

Reply:
Looks good for me.